Developer-Guide.md

```diff
@@ -166,6 +188,41 @@ $ sudo install -o root -g root -m 0640 -D kata-containers.img "/usr/share/kata-c
 $ (cd /usr/share/kata-containers && sudo ln -sf "$image" kata-containers.img)
 ```
 
+## Create an initrd image - OPTIONAL
+### Create a local rootfs for initrd image
+```
+$ cd $GOPATH/src/github.com/kata-containers/osbuilder/rootfs-builder
+$ script -fec 'sudo -E GOPATH=$GOPATH AGENT_INIT=yes USE_DOCKER=true ./rootfs.sh ${distro}'
+```
+`AGENT_INIT` controls if the guest image uses kata agent as the guest `init` process. When you create an initrd image,
+always set `AGENT_INIT` to `yes`.
+
+You can choose `alpine`, `centos`, `clearlinux`, `euleros`, and `fedora` for `${distro}`. If `${distro}` is not set,
+it builds `centos` based rootfs by default.
+
+Optionally, add your custom agent binary to the rootfs as shown in the previous [Add a custom agent
+to the image](#build-a-custom-kata-agent---optional) section, with one additional last step:
```

Comment:
Post creating a local rootfs for initrd image there is no  such '../../agent/kata-agent.service'  file or directory. 
Do we mean "../../agent/kata-agent.service.in" here? The kata-agent.service file gets deleted post building of rootfs for initrd image.

Reply:
For initrd image, we always use the kata agent as `init` process and do not rely on `systemd` so there are no such .service files. Do you see any error without it?

Reply:
Yes, you are right. I've fixed it. PTAL.

Reply:
@bergwolf : This will error out as  ${ROOTFS_DIR}/sbin/init is not a directory. Syntax being: 
install [OPTION]... -t DIRECTORY SOURCE